web/github/connect/index.html

```diff
@@ -31,7 +31,7 @@
     </a>
 
     <table class="title">
-      <td><img class="logo" src="/static/images/github.svg" alt="GitHub" /></td>
```

Comment:
In general we should have one error and success HTML template and have the connection populate them.

Reply:
I agree, I hope to do that once we have more hands on deck :)